presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -170,6 +188,10 @@ public static DistributedQueryRunner createQueryRunner(
                         .setExternalWorkerLauncher(externalWorkerLauncher)
                         .build();
         try {
+            if (addJmxPlugin) {
```

Comment:
Was there any reason to put the JMX plugin as the first one to install? If no specific reasons, it would be better to add it after Hive plugin, to match the arguments order ( you added addJmxPlugin as the last input to this function)